[2023-01-30T08:23:39.038+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: crawl_data.save-to-hdfs scheduled__2023-01-29T17:05:00+00:00 [queued]>
[2023-01-30T08:23:39.051+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: crawl_data.save-to-hdfs scheduled__2023-01-29T17:05:00+00:00 [queued]>
[2023-01-30T08:23:39.052+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-01-30T08:23:39.052+0000] {taskinstance.py:1280} INFO - Starting attempt 4 of 4
[2023-01-30T08:23:39.052+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-01-30T08:23:39.090+0000] {taskinstance.py:1300} INFO - Executing <Task(SSHOperator): save-to-hdfs> on 2023-01-29 17:05:00+00:00
[2023-01-30T08:23:39.095+0000] {standard_task_runner.py:55} INFO - Started process 1451 to run task
[2023-01-30T08:23:39.099+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'crawl_data', 'save-to-hdfs', 'scheduled__2023-01-29T17:05:00+00:00', '--job-id', '1577', '--raw', '--subdir', 'DAGS_FOLDER/crawl_data.py', '--cfg-path', '/tmp/tmp6amzmm00']
[2023-01-30T08:23:39.100+0000] {standard_task_runner.py:83} INFO - Job 1577: Subtask save-to-hdfs
[2023-01-30T08:23:39.181+0000] {task_command.py:388} INFO - Running <TaskInstance: crawl_data.save-to-hdfs scheduled__2023-01-29T17:05:00+00:00 [running]> on host 9be4ecca8506
[2023-01-30T08:23:39.273+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=phukaioh
AIRFLOW_CTX_DAG_ID=crawl_data
AIRFLOW_CTX_TASK_ID=save-to-hdfs
AIRFLOW_CTX_EXECUTION_DATE=2023-01-29T17:05:00+00:00
AIRFLOW_CTX_TRY_NUMBER=4
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-29T17:05:00+00:00
[2023-01-30T08:23:39.275+0000] {ssh.py:133} INFO - Creating ssh_client
[2023-01-30T08:23:39.275+0000] {ssh.py:283} WARNING - No Host Key Verification. This won't protect against Man-In-The-Middle attacks
[2023-01-30T08:23:39.286+0000] {transport.py:1874} INFO - Connected (version 2.0, client OpenSSH_8.2p1)
[2023-01-30T08:23:39.402+0000] {transport.py:1874} INFO - Authentication (password) successful!
[2023-01-30T08:23:39.402+0000] {ssh.py:460} INFO - Running command: /opt/hadoop/bin/hdfs dfs -copyFromLocal -f /home/phukaioh/Documents/intro-data-science/crawler/data/vietnamnet.json /data/vietnamnet/vietnamnet-30-01-2023.json 
[2023-01-30T08:23:42.982+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=crawl_data, task_id=save-to-hdfs, execution_date=20230129T170500, start_date=20230130T082339, end_date=20230130T082342
[2023-01-30T08:23:43.061+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-01-30T08:23:43.084+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
